pkgs/servers/monitoring/prometheus/wireguard-exporter.nix

```diff
@@ -13,6 +13,11 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-NsxGpjuZPpz4gCJRp5IOcfRFh8DTud47nV2bE0/kc2Q=";
 
+  patchPhase = ''
```

Comment:
Please use `postPatch` instead of `patchPhase`, otherwise `patches` are ignored.